### DISCUSSION TITLE
Commit and Transaction support

### DISCUSSION THREAD

**Discussion by sandeep-gh** [tursodatabase/turso#414] [Q&A] [answered]
Would love to contribute to the project, mostly design docs, usage, tests, benchmark and perfs. I had a question which I couldn't answer just by browsing at the code. 
- Is transaction and rollback supported? If not, is it on the roadmap?
- What about commit. Is it at par with sqlite3? Are there test cases for commits?

Thanks. Looking forward.

**Comment by pereman2** [comment, marked as answer]
Hi @sandeep-gh . Any help is welcome, you can take a look at open issues and take whatever you prefer. 

To answer your question about transactions/commits, I'm currently working on it - so that's why you don't see many things in the code related to that :)
